sonali2105.txt

```diff
@@ -0,0 +1 @@
+Sonali Sharma
```

Comment:
This file is to be created inside contributors folder.

Reply:
okay